app/services/api/v3/actors/basic_attributes.rb

```diff
@@ -108,14 +112,23 @@ def importer_summary
           # 1,224 municipalities, or 48% of the soy production municipalities.
           # The main destination of the soy imported by Cargill is China,
           # accounting for 66% of the total.
+
+          initialize_trade_volume_for_summary
           text = summary_of_total_trade_volume('importer')
+          return text if @trade_total_current_year_raw.zero?
+
           text += summary_of_sources('importer')
           text += summary_of_destinations('importer')
           text
         end
 
         def summary_of_total_trade_volume(profile_type)
-          initialize_trade_volume_for_summary
+          if @trade_total_current_year_raw.zero?
+            return "<span>#{@node.name.humanize}</span> \
```

Comment:
I only now realised we're returning html from the backend, this is horrible. I understand this is to support translations; in which case I propose to move the logic of constructing the summary text to frontend from values provided by backend. I'll create an issue for this.